CHANGELOG.md

```diff
@@ -2,12 +2,35 @@
 
 ## 1.5.0
 Improvements:
-- Support variables for Kit.toolchainFile. [#991](https://github.com/microsoft/vscode-cmake-tools/pull/991)[#1056](https://github.com/microsoft/vscode-cmake-tools/issues/1056) [@blakehurd](https://github.com/blakehurd)/[@bobbrow](https://github.com/bobbrow)
+- Support variables for Kit.toolchainFile. [PR #991](https://github.com/microsoft/vscode-cmake-tools/pull/991)[#1056](https://github.com/microsoft/vscode-cmake-tools/issues/1056) [@blakehurd](https://github.com/blakehurd)/[@bobbrow](https://github.com/bobbrow)
+- Implement cmake:hideBuildCommand context option. [PR #1355](https://github.com/microsoft/vscode-cmake-tools/pull/1355) [@tritao](https://github.com/tritao)
+- Add option to set CMAKE_BUILD_TYPE also on multi-config generators. [PR #1393](https://github.com/microsoft/vscode-cmake-tools/pull/1393) [@tonka3000](https://github.com/tonka3000)
+- Detect Clang for MSVC (GNU CLI) kits. [#823](https://github.com/microsoft/vscode-cmake-tools/issues/823) [@omcnoe](https://github.com/omcnoe)
+- GUI support for CMake Tools cache. [#513](https://github.com/microsoft/vscode-cmake-tools/issues/513) [@nieroger](https://github.com/nieroger)
+- Tasks support. [PR #1268](https://github.com/microsoft/vscode-cmake-tools/pull/1268) [@vptrbv](https://github.com/vptrbv)
+- MacBook Pro touchbar support. [#499](https://github.com/microsoft/vscode-cmake-tools/issues/499) [@vptrbv](https://github.com/vptrbv)
+
+Bug Fixes:
+- Set right base_path for variant config files. [PR #1462](https://github.com/microsoft/vscode-cmake-tools/pull/1462) [@leolcao](https://github.com/leolcao)
+- Remove duplicate option start signs. [#1280](https://github.com/microsoft/vscode-cmake-tools/issues/1280) [@dnkbln](https://github.com/dnkbln)
```

Comment:
Did you go back and test this on linux?  I think it needs to be backed out.

Reply:
Yes I did test it again, windows and linux, several different generators... The "--" was really an extra. In all cases I tested, what was following the "--" was also having a switch prefix, either "/" or "-", depending on generator.

Reply:
I'll get soon on MAC, I'll test it there too.

Reply:
Did you test with the build tool args setting set?

Reply:
Maybe the `--` just needs to go after the generator args.

Reply:
It looks like maybe we should just use -j as a cmake option instead of the native build tool switch.  https://cmake.org/cmake/help/v3.18/manual/cmake.1.html#build-a-project